integrations/pinecone/pyproject.toml

```diff
@@ -24,8 +24,7 @@ classifiers = [
 ]
 dependencies = [
   "haystack-ai",
-  "pinecone-client<3", # our implementation is not compatible with pinecone-client>=3
-  # see https://github.com/deepset-ai/haystack-core-integrations/issues/223
+  "pinecone-client>=3"
```

Comment:
I would keep the comment inverting it, like `our implementation is not compatible with pinecone-client <3`